src/mocks/handlers.ts

```diff
@@ -190,6 +192,63 @@ export const handlers = [
     }
   }),
 
+  http.post<
+    PathParams,
+    CatalogueCategoryPropertyMigration,
```

Comment:
I think these schemas 
```suggestion
    AddPropertyMigration,
```